### PR TITLE
Add ability to save multiple responses mid-exercise.

### PR DESCRIPTION
**Pull request by eschlange**
* Multisyllabic exercises now allow for multiple reponses per question.
* Refactored existing sound recording logic.
* Removed unused JS file.

[#106168202]

### CODE REVIEW

File: app/assets/javascripts/picture_deck.js [hunk 1]
```diff
@@ -33,54 +33,77 @@ function postResponse(responseObject, options){
...
+function hideHintAndRecordButtons() {
+  $('#hint-link').addClass("hidden");
+  $('#record-button').addClass("hidden");
+}
+
+function concludeExercise(hintCount, timeStart, response) {
+  $('#next-button').text("Finish");
+  $('.centered').append('<div class="pagination-link-restart"><a id="restart" href="">Practice again</a></div>');
+  $('#next-button, a[href="/"], #restart').unbind();
+  $('#next-button, a[href="/"], #restart').on("click", function(){
```
Comment (michaelwehrley) [inline app/assets/javascripts/picture_deck.js]: FYI, you can chain these as well: 
```
$('#next-button, a[href="/"], #restart')
  .unbind()
  .on("click", function(){
```
if that doesn't work, you can call `.end()` to get back to the base jquery wrapper.

**Comment by michaelwehrley** [comment]
Looks good.  Are there any specs?

**Comment by eschlange** [comment]
No specs, the project is not configured for them...

**Comment by michaelwehrley** [comment]
:+1: merge when ready.

**Pull request merged**